src/RequestHandler/RequestManager.js

```diff
@@ -0,0 +1,26 @@
+const {HandlerFactory} = require('./HandlerFactory');
+function RequestManager () {
+  const handlersList = {};
+
+  async function createReqHandler(dataFromParser) {
+    for (const [name,data] of Object.entries(dataFromParser.servers)) {
+      handlersList[parseInt(name, 10)] = await HandlerFactory(data , dataFromParser.PublishOperations);
```

Comment:
You need to handle the potential undefined handler here as well, otherwise, you will have unintentional crashes once you start the operations.

Reply:
ok

Reply:
This is still not handled. It will throw unexpected exceptions if we are keeping an undefined value in the `handlersList`.

Reply:
ok i changed it and now if the protocol is not one of the known values it does not assign it to `handlersList`

Reply:
ok changed it.